performance-tests/topo.py

```diff
@@ -7,18 +7,18 @@
 
 class Single(Topo):
 
-    def __init__(self):
+    def __init__(self, no_hosts):
         Topo.__init__(self)
 
         sw = self.addSwitch('s1', dpid='000000000000001')
 
         internet = self.addHost('i0', mac='00:00:00:00:00:02', ip='10.0.0.40')
         self.addLink(sw, internet)
-
-        for i in range(2):
+        assert no_hosts < 254
+        for i in range(no_hosts):
             h = self.addHost(
                 'h%d' % i, mac='00:00:00:11:11:%02x' % i, ip=('10.0.0.1%d' % i))
```

Comment:
this (IP address) wont work if i is >99.
change so it can be < 250.
also means changing any static IP addresses.